verifier.py

```diff
@@ -27,30 +27,12 @@
     run_info = json.load(file)
 
 
-# Takes a URL and produces its relative file name.
-def get_path_from_url(url):
-    # Remove http://domain
-    tail = url.replace(base_urls[0], '') + 'index.html'
-    path = MIRROR + tail
-    return path
-
-
-# Creates a dictionary with filename : URL for all the URLs found by the crawler in the API
-def generate_page_dictionary(self):
```

Comment:
@zamattiac Then why is generate_page_dictionary not showing up in the page superclass in the diff?